Smash_Combos.Persistence/Migrations/20200816160856_UpdatedForModeration.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Smash_Combos.Persistence.Migrations
+{
+    public partial class UpdatedForModeration : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
```

Comment:
You should probably do this after creating the new column, and migrate over any existing admins so that they're still admins in the new column